test/performance_tests.jl

```diff
@@ -29,11 +30,18 @@ function bench_press(
     min_allocs = 0.0,
 )
     println("Testing ", "$foo")
-    # Calling foo once before benchmarking
-    # to make sure compile time is not included in the benchmark
-    foo(args...)
-    # Benchmark foo
-    trail = BT.@benchmark $foo($args...)
+    if typeof(args) == String
```

Comment:
I'm a bit confused, by branch you're referring to the if-else statements? And if I did that, would the following commands work if the arguments aren't strings? I made an if-else statement because `args...` was breaking my string argument apart into many chars so I don't think I would want it to have the same interface as `args...`. But I think I'm misunderstanding what you are saying

Reply:
Yes, branch means the if/else statement in this context. What this line does is that if `args` is a `String` it will package it into a `tuple` with the string inside. (If its not a `String` it will do nothing) And then `Tuple{String}` plays nicely with the `foo(args...)` syntax